services/wallet/activity/activity.go

```diff
@@ -613,6 +617,7 @@ func getActivityEntries(ctx context.Context, db *sql.DB, addresses []eth.Address
 		} else if multiTxID.Valid {
 			mtInAmount, mtOutAmount := getMtInAndOutAmounts(dbMtFromAmount, dbMtToAmount)
 			activityType := multiTransactionTypeToActivityType(transfer.MultiTransactionType(dbMtType.Byte))
+			fmt.Println("getMtInAndOutAmounts", multiTxID.Int64, activityType, mtInAmount, mtOutAmount)
```

Comment:
I will remove these leftovers in my token PR and move the amounts helpers after the main `getActivityEntries `. The query should be close to the implementation for reference purposes. WDYT?

Reply:
Oops! Sure, thanks!